style/ruby/README.md

```diff
@@ -3,47 +3,51 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
-* Avoid multiple assignments per line (`one, two = 1, 2`).
-* Avoid organizational comments (`# Validations`).
+* Avoid conditional modifiers (lines that end with conditionals). 36491dbb9
```

Comment:
Yeah, maybe leave them as-is and we ask GitHub to add the auto-linking to MD docs.